android/src/main/java/com/pauldemarco/flutterblue/FlutterBluePlugin.java

```diff
@@ -517,6 +523,15 @@ public boolean onRequestPermissionsResult(
         return false;
     }
 
+    private void removeBond(BluetoothDevice device) {
+        try {
+            Method m = device.getClass().getMethod("removeBond",null);
+            m.invoke(device, null);
```

Comment:
Can you get rid of the reflection calls?
This is not only comparable slow, it also breaks if on release the app enables proguard (or R8 nowadays), which is enabled by default (and should!!!)

Reply:
OK. Thank You

Reply:
Leo,andretietz: 
device.connect(isBond: true)
device.disconnect(isRemoveBond: true)
does these codes works in 0.6.3?